src/cljs/pyregence/pages/near_term_forecast.cljs

```diff
@@ -34,25 +33,6 @@
             [reagent.core                                        :as r]
             [reagent.dom                                         :as rd]))
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; Spec
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
-(s/def ::clear-point?  boolean?)
-(s/def ::filter        string?)
-(s/def ::filter-set    set?)
-(s/def ::geoserver-key keyword?)
-(s/def ::opt-label     string?)
-(s/def ::units         string?)
-(s/def ::z-index       int?)
-(s/def ::layer-config  (s/keys :req-un [::opt-label (or ::filter ::filter-set)]
-                               :opt-un [::clear-point? ::geoserver-key ::units ::z-index]))
-(s/def ::layer-path    (s/and (s/coll-of keyword? :kind vector? :min-count 2)
-                              (s/cat :forecast #{:fuels :fire-weather :fire-risk :active-fire :psps-zonal}
-                                     :second   #(or (= % :params)
-                                                    (= % :underlays))
-                                     :etc      (s/+ keyword?))))
```

Comment:
Why don't we just add another optional spec option to the `::layer-path` spec? We can add something like:
```clojure
(s/def ::default-option keyword?)
```
and then update `::layer-config` to include `::default-option` in the `:opt-un` section.

Reply:
Actually, ignore this. I think I've come up with an easier way to make this update that doesn't involve updating the DB at all. I'll push a commit with my thoughts shortly.